Description: Upstream changes introduced in version 1.4.0-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 clam-networkeditor (1.4.0-1) unstable; urgency=low
 .
   * Initial release (Closes: #517882)
 .
 The person named in the Author field signed this changelog entry.
Author: TANIGUCHI Takaki <takaki@debian.org>
Bug-Debian: http://bugs.debian.org/517882

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- clam-networkeditor-1.4.0.orig/src/widgets/EmbededWidgets.hxx
+++ clam-networkeditor-1.4.0/src/widgets/EmbededWidgets.hxx
@@ -9,8 +9,9 @@ namespace CLAM
 
 class EmbededWidgetCreatorBase
 {
-private:
+public:
 	typedef std::string Key;
+private:
 	typedef std::map<Key, EmbededWidgetCreatorBase*> Creators;
 	static Creators & creators();
 protected:
